tasmota/berry/include/be_gpio_defines.h

```diff
@@ -356,7 +356,8 @@ const be_const_member_t lv_gpio_constants[] = {
     { "ZIGBEE_RST", (int32_t) GPIO_ZIGBEE_RST },
     { "ZIGBEE_RX", (int32_t) GPIO_ZIGBEE_RX },
     { "ZIGBEE_TX", (int32_t) GPIO_ZIGBEE_TX },
-
```

Comment:
The keys should be in alphabetical order. Anyways this file is generated and should not be changed manually.

Reply:
Was not aware of, good to know